src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 100;
+    private int size;
+    private K[] keys;
+    private V[] values;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_ITEMS_NUMBER];
+        values = (V[]) new Object[MAX_ITEMS_NUMBER];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if (checkEquals(keys[i], key)) {
+                values[i] = value;
+                return;
+            }
+        }
+        keys[size] = key;
+        values[size] = value;
+        size++;
     }
 
     @Override
     public V get(K key) {
-        return null;
+        for (int i = 0; i < size; i++) {
+            if (checkEquals(keys[i], key)) {
+                return values[i];
+            }
+        }
+        return null; //throw new RuntimeException("The given key does not exist in storage");
+        //  Don’t return null from a method.
+        //  Returning null from a method is a bad practice говороли вони ;D
     }
 
     @Override
     public int size() {
-        return -1;
+        return size;
+    }
+
+    public boolean checkEquals(K key1, K key2) {
```

Comment:
Also, should this method be public?
https://stackoverflow.com/a/3874378
```suggestion
     private boolean checkEquals(K key1, K key2) {
```